src/elm/Pages/Build/View.elm

```diff
@@ -755,9 +756,12 @@ viewBuildHistory now timezone page org repo builds limit =
         case builds of
             RemoteData.Success blds ->
                 if List.length blds > 0 then
-                    ul [ class "build-history", class "-no-pad", Util.testAttribute "build-history" ] <|
-                        List.indexedMap (viewRecentBuild now timezone org repo buildNumber) <|
-                            List.take limit blds
+                    div [ class "build-history" ]
+                        [ p [ class "build-history-title" ] [ text "Recent Builds" ]
+                        , ul [ class "-no-pad", Util.testAttribute "build-history" ] <|
```

Comment:
since we moved the `-no-pad` style to the `ul` and removed the `.build-history.-no-pad` class, can we remove it from the element?
```suggestion
                        , ul [ Util.testAttribute "build-history" ] <|
```